doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,90 @@
+# Advocacy Ambassador program
+
+The project aims to support and encourage developers and developer
+advocates to promote Node.js as a project and to help communicate
+key messages to their constituencies.
+
+The ambassador program does that by:
+
+* Maintaining an up-to-date set of messages that it would like ambassadors
+  to promote. These will be documented in the nodejs/ambassadors repository.
+* Maintaining an up-to-date set of topics on which additional content
+  would benefit the community. These will be documented in the
+  nodejs/ambassadors repository.
+* Providing a space for collaborators to share new topics/features they
+  are working on, when they would be happy for ambassadors to help share and get
+  early feedback.
+* Helping ambassadors promote content that aligns with the
+  messages and topics defined through social media and other channels.
+* Advocating for ambassadors to be part of the OpenJS speakers bureau, even if the
+  ambassador is not otherwise an active member of the project itself.
+
+## Ambassadors nominations
+
+Any member of the Node.js organization may nominate an ambassador.
+The Technical Steering Committee (TSC) will review/approve ambassadors.
+
+In the initial phase of the program:
+
+* there will be a maximum of 4 ambassadors
+* the term will be for the calendar year. After a year ambassadors automatically
+  drop out of the ambassador list and must be re-nominated
+* ambassadors will be documented on the nodejs/ambassadors README.md
+
+The TSC may choose to remove an ambassador if things are not working
+out in terms of the content promoted and alignment with the project.
+
+Members of the Node.js organization can nominate ambassadors by opening
+an issue in: TBD.
```

Comment:
This shouldn't land until `TBD` is replaced, right?

Reply:
@Trott replace the TBD.